README.md

```diff
@@ -536,6 +536,9 @@ Thirty-sixth Conference on Neural Information Processing Systems (NeurIPS), 2022
 - <a name="Elshair22opteng"></a>El Shair, Z., Rawashdeh, S.A.,  
 *[High-temporal-resolution event-based vehicle detection and tracking](https://doi.org/10.1117/1.OE.62.3.031209)*,  
 Optical Engineering, 2022. [dataset](http://sar-lab.net/event-based-vehicle-detection-and-tracking-dataset/).
+- <a name="Guillen-Garcia22"></a>Guillen-Garcia, J., Palacios-Alonso, D., Cabello, E., Conde, C.,  
+*[Unsupervised adaptive multi-object tracking-by-clustering algorithm with a bio-inspired system](https://doi.org/10.1109/ACCESS.2022.3154895)*,  
+IEEE Access, 2022. [PDF](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9721918)
```

Comment:
no need for ieeexplore link because it is pointed at by the doi link.

Reply:
Fixed, thanks.